ktx-mapbox-maps/src/main/java/com/mapbox/mapboxsdk/plugins/maps/Query.kt

```diff
@@ -27,9 +26,7 @@ inline fun MapboxMap.queryRenderedFeatures(latLng: LatLng,
  * @param layerIds    optionally - only query these layers
  * @return the list of feature
  */
-inline fun MapboxMap.queryRenderedFeatures(latLng: LatLng,
-                                           filter: Expression?,
-                                           vararg layerIds: String): List<Feature> {
+fun MapboxMap.queryRenderedFeatures(latLng: LatLng, filter: Expression?, vararg layerIds: String): List<Feature> {
```

Comment:
What if we make `filter` and `layerIds` default to `null`? This should suffice for both of methods and add an option of passing only `latLng`.